### PR TITLE
Feature EPL-1730: First model of the widged structure component

### PR DESCRIPTION
**Pull request by Sankku**
Storybook broke in drawer component (The build is fixed just have to adaptate the mocks or directly delete the drawer from storybook)

image

### CODE REVIEW

**Review by leandroallemandismf — APPROVED** [review]

File: packages/ComponentLibrary/src/components/Table/styles.ts [hunk 1]
```diff
@@ -88,6 +89,9 @@ export const tableStyles: Record<string, SxProps<Theme>> = {
     backgroundSize: 'cover',
     backgroundPosition: 'center',
     backgroundRepeat: 'no-repeat',
+    overflowY: 'auto',
+    display: 'flex',
+    height: '48.5rem',
```
Comment (LuuchoRocha) [inline packages/ComponentLibrary/src/components/Table/styles.ts:94]: 👀

File: packages/ComponentLibrary/src/components/Table/styles.ts [hunk 2]
```diff
@@ -182,14 +186,45 @@ export const sx: { [key: string]: SxProps<Theme> } = {
   container: {
     gridContainer: '1rem',
   },
+  widgetContainer: {
+    color: theme.palette.baselineColor.neutral[0],
+    height: '100%',
+    minHeight: '8rem',
+    background: 'red',
```
Comment (LuuchoRocha) [inline packages/ComponentLibrary/src/components/Table/styles.ts:193]: is this correct?

File: packages/storybook/src/stories/Components/Table/mockWidget.tsx [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+    '#2186eb',
+    '#0967d2',
+    '#0552b5',
+    '#03449e',
+    '#01337d',
+    '#002159',
+  ],
+};
+
+// Función para asegurar el contraste
```
Comment (sjcuello) [inline packages/storybook/src/stories/Components/Table/mockWidget.tsx:46]: English comment

**Review by sjcuello — APPROVED** [review]
just fix the comment

**Pull request merged**